en/release-notes.md

```diff
@@ -25,6 +24,13 @@ The official Python client library for MAAS is available in the Ubuntu 18.04
 LTS package archive or you can download the source from:
 [https://github.com/maas/python-libmaas/releases](https://github.com/maas/python-libmaas/releases)
 
+## 2.4.0 (beta3)
+
+This release addresses various UI and backend issues, improving the stability
+of MAAS 2.4. See below for a list of all 31 bug fixes in this release:
```

Comment:
Sounds odd. Should it be "fixed"?

Reply:
doh, thanks - good catch. I'll reword this.